.circleci/config.yml

```
@@ -208,10 +208,10 @@ jobs:
             mkdir /logs
             # conda update -y conda > /logs/conda.txt 2>&1
             # Create and activate conda environment
-            conda create -y --name esmvaltool
+            conda create -y --name esmvaltool 'python=3.9'
```

```suggestion
            conda create -y --name esmvaltool -c conda-forge 'python=3.9'
```
or it's going to create an environment full of packages from the defaults channel that then need to be uninstalled in the next step

---

very good point! Overlooked it :eyes: